tests/test_cli.py

```diff
@@ -1,19 +1,40 @@
-from click.testing import CliRunner
+from unittest.mock import ANY, mock_open
```

Comment:
`pytest` is generally the framework of choice for our tests - only the really really old ones still use `unittest` if we can avoid leaning on it further that would be ideal :pray: